tests/test_utils_data_cleaning_anywin_oscars.py

```diff
@@ -58,19 +58,16 @@ def test_clean_validate_anywin_oscars(self):
         ]
 
         for data, exp_output in dfs_to_test:
-            flag = False
 
-            input_df =pd.DataFrame(data)
-            cleaned = cleaner_aw_oscars._clean(input_df)
-            exp_output_df = pd.DataFrame(exp_output)
-            clean_flag = cleaned.equals(exp_output_df)
-            self.assertTrue(clean_flag)
-            
-            cleaner_aw_oscars._validate(cleaned)
+            try:
+                input_df =pd.DataFrame(data)
+                cleaned = cleaner_aw_oscars._clean(input_df) # pylint: disable=W0212
+                exp_output_df = pd.DataFrame(exp_output)
+                self.assertTrue(cleaned.equals(exp_output_df))
+                cleaner_aw_oscars._validate(cleaned) # pylint: disable=W0212
 
-            flag = True
-
-            self.assertTrue(flag)
+            except Exception: # pylint: disable=W0703
```

Comment:
Why wrap in a try except? What was failing?

Reply:
Was just changing the syntax to something closer to this: https://stackoverflow.com/questions/4319825/python-unittest-opposite-of-assertraises

instead of using the flags and self.assertTrue(flag) statements like in my previous commits. I'll just change it back to how I had it before.

Reply:
Ah okay. I was making that comment for tests where you expect them to fail - a try/except is a good choice there. For code that you expect to run, you can just run the test on it. 

This is a style nit though - if the tests pass go ahead and merge them.